Mobile Buy SDK Sample Apps/Mobile Buy SDK tvOS Sample/Mobile Buy SDK tvOS Sample/ProductItem.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ProductItem.swift
+//  Mobile Buy SDK tvOS Sample App
+//
+//  Created by Shopify.
+//  Copyright (c) 2016 Shopify Inc. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import Foundation
+import Buy
+
+class ProductItem: NSObject {
```

Comment:
This class is also used mainly to abstract model objects from views and view controllers. The class is named `ProductItem` because the class represents a product and holds the contents needed to present a product. Currently since we only present product images to the page view controller, we only need this class to store product images but the user can easily add things like product price, title, and more to this class and then use it in the view controller